Cargo.lock

```diff
@@ -1789,6 +1785,21 @@ dependencies = [
  "wasm-bindgen",
 ]
 
+[[package]]
+name = "jsonwebtoken"
+version = "9.2.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "5c7ea04a7c5c055c175f189b6dc6ba036fd62306b58c66c9f6389036c503a3f4"
+dependencies = [
+ "base64 0.21.5",
+ "js-sys",
+ "pem",
+ "ring 0.17.7",
```

Comment:
note to self that jsonwebtoken's requirment for ring and serde is not the same as what's already in here, might need some version bumping somewhere

Reply:
Closing the loop here, looks like when we bumped our version on actix, we pulled ring 0.17 - and the only dependant of ring 0.16, which keeps it in our dependency tree is `hawk`. Looking at `hawk`'s main, [they already have the right version of `ring` on their main branch](https://github.com/taskcluster/rust-hawk/blob/41f435cbaf31994f1f655dcefa7434079d221fa4/Cargo.toml#L29) so they just need to cut a release to get rid of the duplicate dependency

Reply:
FWIW I asked the taskcluster folks if they could cut a new release for `hawk` so we can eliminate that dependency

Reply:
https://github.com/mozilla-services/syncstorage-rs/pull/1516 fixes this!